README.md

```diff
@@ -40,3 +40,19 @@ use http://localhost:5000/ in you browser to see the application.
 If new libraries or packages were added then:
 
     $ vagrant provision
+
+
+## Production deployment
```

Comment:
Maybe add Ubuntu as a requirement for this setup? Others should work, but requires some tinkering with the setup I think.